baseapp-auth/baseapp_auth/templates/users/emails/new-superuser-notification-email.html.j2

```diff
@@ -3,12 +3,12 @@
 {% block content %}
 <tr style="font-family: 'Helvetica Neue',Helvetica,Arial,sans-serif; box-sizing: border-box; font-size: 14px; margin: 0;">
     <td class="content-block" style="font-family: 'Helvetica Neue',Helvetica,Arial,sans-serif; box-sizing: border-box; font-size: 14px; vertical-align: top; margin: 0; padding: 0 0 20px;" valign="top">
-        Hello Superusers,
+        {% trans %}Hello Superusers,{% endtrans %}
     </td>
 </tr>
 <tr style="font-family: 'Helvetica Neue',Helvetica,Arial,sans-serif; box-sizing: border-box; font-size: 14px; margin: 0;">
     <td class="content-block" style="font-family: 'Helvetica Neue',Helvetica,Arial,sans-serif; box-sizing: border-box; font-size: 14px; vertical-align: top; margin: 0; padding: 0 0 20px;" valign="top">
-        Please be notified that, <b>{{ assignee }}</b> is a superuser now. Made by <b>{{ assigner }}</b>.
+        {% trans %}Please be notified that,{% endtrans %} <b>{{ assignee }}</b> {% trans %}is a superuser now. Made by {% endtrans %}<b>{{ assigner }}</b>.
```

Comment:
Lets have this as one  `{% trans %}` instead of breaking into multiples. I see on the other templates you used 1  `{% trans %}` for the whole line.